ssg/build_cpe.py

```diff
@@ -104,6 +106,46 @@ def get_cpe_name(self, cpe_id):
     def get_product_cpe_names(self):
         return [ cpe.name for cpe in self.product_cpes.values() ]
 
+    def parse_platform_definition(self, platforms):
+        # let's construct the platform id
+        id_list = [self._convert_platform_to_id(platform) for platform in platforms]
+        id = "_or_".join(id_list)
+        id = "cpe_platform_" + id
+        platform = CPEALPlatform(id)
+        # add initial test
+        initial_test = CPEALTest(operator="OR", negate="false")
+        platform.add_test(initial_test)
+        for platform_line in platforms:
+            initial_test.add_object(self.parse_platform_line(platform_line))
+        return platform
+
+    def parse_platform_line(self, platform_line):
```

Comment:
This function assumes that a `platform_line` has a single operator.
It should be possible to have  `platform_line` like 'A & B & !C`.